cypress/features/regression/flatTable.feature

```diff
@@ -0,0 +1,87 @@
+Feature: FlatTable component
```

Comment:
feature file needs to be moved to separate directory `Test`

Reply:
fixed